src/Options/Options.hpp

```diff
@@ -356,37 +381,37 @@ void Options<OptionsList>::parse(const std::string& options) {
 
 template <typename OptionsList>
 void Options<OptionsList>::parse(const Option_t& options) {
-  context_ = options.Context();
-  parse(options.Node());
+  context_ = options.context();
+  parse(options.node());
 }
 
 template <typename OptionsList>
-void Options<OptionsList>::parse(const YAML::Node& options) {
-  if (not(options.IsMap() or options.IsNull())) {
+void Options<OptionsList>::parse(const YAML::Node& node) {
+  if (not(node.IsMap() or node.IsNull())) {
     PARSE_ERROR(context_,
-                "'" << options << "' does not look like options.\n"
+                "'" << node << "' does not look like options.\n"
                 << help());
   }
-  for (auto it = options.begin(); it != options.end(); ++it) {
-    const auto key = it->first.as<std::string>();
-    const auto value = it->second;
+  for (const auto& kv : node) {
```

Comment:
while I've used kv for the range-based key-value pair, others  (  @gsb76 , @osheamonn ) have voiced other opinions